composite/bitcoin-json-rpc/src/adapter.ts

```diff
@@ -0,0 +1,36 @@
+import JSONRPC from '@chainlink/json-rpc-adapter'
+import { Config, ExecuteWithConfig, ExecuteFactory, AdapterRequest } from '@chainlink/types'
+import { Validator, Requester } from '@chainlink/external-adapter'
+import { makeConfig, DEFAULT_ENDPOINT } from './config'
+
+const inputParams = {
+  endpoint: false,
+}
+
+const convertEndpoint: { [key: string]: string } = {
+  height: 'headers',
+}
+
+// Export function to integrate with Chainlink node
+export const execute: ExecuteWithConfig<Config> = async (request: AdapterRequest) => {
+  const validator = new Validator(request, inputParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  let endpoint = validator.validated.data.endpoint || DEFAULT_ENDPOINT
+
+  const jsonrpcRequest: AdapterRequest = { id: jobRunID, data: { method: 'getblockchaininfo' } }
+  const response = await JSONRPC.execute(jsonrpcRequest)
```

Comment:
You could include the incoming `inputRequest` into the `jsonrpcRequest` so you give this adapter the option of having the input params of the `json-rpc` adapter.  Something like:
```suggestion
const response = await JSONRPC.execute({
    ...request,
    data: { ...request.data, method: 'getblockchaininfo' },
  })
```